### DISCUSSION TITLE
Need Confirmation regarding using Path while using Slint in microcontrollers like rp2040

### DISCUSSION THREAD

**Discussion by sundaritisthatis** [slint-ui/slint#4314] [Q&A] [answered]
I have checked the documentation - need confirmation on what type of renderer is used by default in microcontroller usage.

if it is available - when can we expect the path support on this

**Comment by tronical** [comment, marked as answer]
In Microcontrollers by default the software renderer is used. We're tracking support for `Path` in the software renderer under https://github.com/slint-ui/slint/issues/4178 .
